ArduCopter/defines.h

```diff
@@ -54,7 +54,7 @@ enum tuning_func {
     TUNING_DECLINATION =                38, // compass declination in radians
     TUNING_CIRCLE_RATE =                39, // circle turn rate in degrees (hard coded to about 45 degrees in either direction)
     TUNING_ACRO_YAW_KP =                40, // acro controller's P term.  converts pilot input to a desired roll, pitch or yaw rate
-    TUNING_RANGEFINDER_GAIN =           41, // rangefinder gain
+    TUNING_RANGEFINDER_GAIN =           41, // unused
     TUNING_EKF_VERTICAL_POS =           42, // unused
     TUNING_EKF_HORIZONTAL_POS =         43, // unused
     TUNING_EKF_ACCEL_NOISE =            44, // unused
```

Comment:
```suggestion
```
If we just delete all of these then we can be sure nowhere in the code is trying to use them. 

And the param values here:
https://github.com/ArduPilot/ardupilot/blob/9d32e481c21a950d15482a19b4bef4d349e177e1/ArduCopter/Parameters.cpp#L349

Reply:
Good catch on the Parameter description!
I have not removed the lines in the defines.h file as that is a style change and best off left as a separate commit from people with style.